pkg/cmsis-dsp/Makefile

```diff
@@ -8,6 +8,6 @@ CFLAGS += -Wno-strict-aliasing
 .PHONY: all
 
 all: git-download
-	$(MAKE) -C $(PKG_BUILDDIR)
+	"$(MAKE)" -C $(PKG_BUILDDIR)
```

Comment:
While not necessarily related to this PR, I suspect the `$(PKG_BUILDDIR)` variables should also be quoted here and in the other makefiles. This looks like it breaks when there is whitespace in `PKG_BUILDDIR`.

Reply:
True... We could open an issue and tag it for newbies?